### PR TITLE
Asset Meta styling

### PR DESCRIPTION
**Pull request by joyrostie**
Styling object that defines CSS styles for next components:
AssetMeta
DocumentTable
AssetEventsPanel

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-272.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-272.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-272.gearboxjs.preview.cogniteapp.com

File: src/components/AssetMeta/AssetMeta.tsx [hunk 1]
```diff
@@ -107,15 +118,78 @@ export class AssetMeta extends React.Component<AssetMetaProps, AssetMetaState> {
     contentLen > 0 ? (
       <span>{tab}</span>
     ) : (
-      <span style={{ color: '#9b9b9b' }}>({tab})</span>
+      <span
```
Comment (xitman78) [inline src/components/AssetMeta/AssetMeta.tsx]: Please create styled-component from `span` with `color: #9b9b9b`

Comment (joyrostie) [reply src/components/AssetMeta/AssetMeta.tsx]: Replaced with styled-component

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-272.gearboxjs.preview.cogniteapp.com

File: src/components/AssetMeta/AssetMeta.tsx [hunk 2]
```diff
@@ -130,41 +204,21 @@ export class AssetMeta extends React.Component<AssetMetaProps, AssetMetaState> {
 
     return asset ? (
       <>
-        <h3>{asset.name ? asset.name : asset.id}</h3>
-        {asset.description && <p>{asset.description}</p>}
+        <Header style={styles && styles.header}>
```
[the remark this answers is not in the document]
Comment (joyrostie) [reply src/components/AssetMeta/AssetMeta.tsx]: replaced with `div` and removed styled-component

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-272.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-272.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-272.gearboxjs.preview.cogniteapp.com

File: src/components/AssetMeta/components/AssetEventsPanel.test.tsx [hunk 3]
```diff
@@ -12,13 +12,35 @@ describe('AssetEventsPanel', () => {
...
   });
+  it.each`
+    inlineStyle                              | expected
+    ${'style="width: 80%;"'}                 | ${true}
+    ${'style="background: rgb(0, 255, 0);"'} | ${true}
+    ${'style="font-style: italic;"'}         | ${true}
+  `(
+    'should render component with passed inline styles',
+    ({ inlineStyle, expected }) => {
+      const EventPanel = mount(
```
Comment (mrtraser) [inline src/components/AssetMeta/components/AssetEventsPanel.test.tsx]: `EventPanel` should be written in camelCase, like `eventPanel` or just `wrapper`

File: src/components/AssetMeta/components/AssetEventsPanel.test.tsx [hunk 4]
```diff
@@ -12,13 +12,35 @@ describe('AssetEventsPanel', () => {
...
+        <AssetEventsPanel events={EVENTS} styles={ASSET_META_STYLES.events} />
+      );
+      const containerTable = EventPanel.find('.ant-table-wrapper')
+        .last()
+        .html();
```
Comment (mrtraser) [inline src/components/AssetMeta/components/AssetEventsPanel.test.tsx]: It's not a good way to check if styles passing to expected elements. Suggestion – use `.prop('style')` on expected element to get `style` value and compare with passed to element.

File: src/components/AssetMeta/stories/AssetMeta.stories.tsx [hunk 5]
```diff
@@ -160,4 +167,61 @@ storiesOf('AssetMeta/Examples', module)
...
       },
     }
+  )
+  .add(
+    'Custom Styles',
+    () => {
+      return (
+        <AssetMeta
+          assetId={123456}
+          styles={{
```
[the remark this answers is not in the document]
Comment (joyrostie) [reply src/components/AssetMeta/stories/AssetMeta.stories.tsx]: Replaced with mock

File: src/components/AssetMeta/components/AssetEventsPanel.test.tsx (hunk 3 above)
Comment (joyrostie) [reply src/components/AssetMeta/components/AssetEventsPanel.test.tsx]: naming fixed

File: src/components/AssetMeta/components/AssetEventsPanel.test.tsx (hunk 4 above)
Comment (joyrostie) [reply src/components/AssetMeta/components/AssetEventsPanel.test.tsx]: Test redesigned

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-272.gearboxjs.preview.cogniteapp.com

**Pull request merged**